projects/ngrx.io/tools/transforms/angular-api-package/index.js

```diff
@@ -33,7 +33,6 @@ module.exports = new Package('angular-api', [basePackage, typeScriptPackage])
   .processor(require('./processors/computeStability'))
   .processor(require('./processors/removeInjectableConstructors'))
   .processor(require('./processors/processPackages'))
-  .processor(require('./processors/processNgModuleDocs'))
```

Comment:
Removed because otherwise we'll get an error during the build, e.g.

```ts
7:33:20 PM: error:   "component/LetDirective" has no @ngModule tag. Docs of type "directive" must have this tag. - doc "component/LetDirective" (directive)  - from file "component/src/let/let.directive.ts" - starting at line 34, ending at line 227
7:33:20 PM: error:   Error running processor "processNgModuleDocs":
```